screens/searching_request.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { StyleSheet, Text, View, Dimensions, Image} from 'react-native';
+
+export default class Searching extends React.Component {
+  render() {
+    return (
+        <View style={styles.container}>
+            <Image source ={require('./../assets/images/shopping-bag.png')} style={styles.bag}/>
+            <Image source ={require('./../assets/images/personicon.png')} style={styles.person}/>
+            <View style={styles.peachBox}/>
+            <View style={styles.whiteBox}>
+                <Text style={styles.header}>Searching for Request</Text>
+                <Image source ={require('./../assets/images/loading.png')} style={styles.loading}/>
+            </View>
+        </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
```

Comment:
Stop using `absolute` style. In other words, don't use the properties `top`, `bottom`, `left`, and `right`. Think about how different screens will display these containers